pkg/gitreceive/git/sha.go

```diff
@@ -0,0 +1,37 @@
+package git
+
+import (
+	"fmt"
+	"regexp"
+)
+
+const (
+	// this constant represents the length of a shortened git sha - 8 characters long
+	shortShaIdx = 8
+	fullShaLen  = 40
```

Comment:
As far as I can see this isn't actually used. Was there intention to use this somewhere?